src/js/main.js

```diff
@@ -42,6 +42,24 @@ export function init(el, context, config) {
             hideAboutBtn.classList.remove('docs--show-about');
         };
 
+
+        // Show the trailer on click #docs__playTrailer
+        const $showTrailerAll = builder.querySelectorAll('.docs__shows-trailer');
+        [].forEach.call($showTrailerAll, function($showTrailer) {
```

Comment:
Cool, I have updated the open showTrailer section — but left the closeTrailer section as a forEach as there are multiple elements that close the trailer?